go/chat/convsource.go

```diff
@@ -216,6 +216,7 @@ func (s *RemoteConversationSource) Pull(ctx context.Context, convID chat1.Conver
 		Reason:         reason,
 	}
 	boxed, err := s.ri().GetThreadRemote(ctx, rarg)
+	// XXX rl is never used
```

Comment:
Can just remove this, I just left it from the refactor of this stuff.

Reply:
done